libraries/CAN_comms/Inc/CAN_comms.h

```diff
@@ -11,7 +11,7 @@
 
 
 #include <stdint.h>
-#include "cmsis_os2.h"
+#include "cmsis_os.h"
```

Comment:
Why are we changing this in this PR?

Reply:
Never mind, just saw from your commit history that this change allows CAN_comms to build. Lgtm! Thanks Kyle!